File: fixtures/elixir/post_with_data_binary.ex
request = %HTTPoison.Request{
  method: :post,
  url: "http://localhost:28139/post",
  options: [],
  headers: [
    {~s|Content-Type|, ~s|application/x-www-form-urlencoded|},
  ],
  params: [],
  body: ~s|{"title":"china1"}|
}

response = HTTPoison.request(request)

File: test/fixtures/ruby-httparty/post_with_data_binary.rb
require 'httparty'

url = 'http://localhost:28139/post'
headers = {
  'Content-Type': 'application/x-www-form-urlencoded',
}
body = '{"title":"china1"}'
res = HTTParty.post(url, headers: headers, body: body)
